.github/workflows/release-python.yml

```diff
@@ -0,0 +1,115 @@
+name: Release Python
+
+on:
+  push:
+    tags:
+      - python/v*
+
+jobs:
+  macos:
+    runs-on: macos-11
+    strategy:
+      matrix:
+        python-version: ['3.7', '3.8', '3.9', '3.10']
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Install Rust
+        uses: actions-rs/toolchain@v1
+        with:
+          toolchain: stable
+          target: aarch64-apple-darwin
+          profile: minimal
+          default: true
```

Comment:
Either this should be when `matrix.python-version != '3.7'` or there should be a separate step for building an ARM64 wheel for 3.7.

Reply:
3.7 doesn't support apple silicon so I can try and make this conditional. Maybe you have a clever way to do it, but I can't find a way to add conditionals to action inputs.

Reply:
Wouldn't this work?

```suggestion
      - name: Install Rust
        uses: actions-rs/toolchain@v1
        if: matrix.python-version != '3.7'
        with:
          toolchain: stable
          target: aarch64-apple-darwin
          profile: minimal
          default: true
```

Reply:
That seems to work, thank you.